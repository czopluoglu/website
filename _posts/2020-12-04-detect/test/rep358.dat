11010000011010101111
10011011011010100011
01110111111001101111
01010101010100000110
10110101101011101010
10011101111011011001
11011000111011010110
10001001001101111111
10111101101010111001
00100101101110110011
01111110111110000101
11111101111101110011
00000101101000011011
00010000001000001010
00000000000000000000
10101101001101110111
00000011100000000010
01000000111010100100
10111101011011101111
01110101111111111101
01100001111100100101
11011001111000111011
11111111110111011111
01010000000001000011
10100101101000011010
11010011110010100110
01011000010101101110
01111101000010110010
10101111111100001111
11111101111001110110
00001111111000011110
01110100101000001100
10000001111010001010
10010101010011110000
11110001011101000110
11111011001110011111
11111101110100001100
10111101110100101111
10010101101001000000
01100001111110111111
10100001001101110111
10100100101100010111
00100000110000100010
00001000101111111111
11110001110010111010
00001100111110110110
10101001111011011001
10000000000000000000
10000000000010000001
10001100011101111000
10101010111010110011
01010010100110111010
01001000000110111111
11111111111111111010
11000001011111111110
11011111100000100101
00010001010110000010
10010101001010001111
11101010111111110111
11011110011100010111
01011111111001000110
01011110010010101100
01100001110111101011
11111111100010101011
01010001101001011010
10001001100011100111
10000010111111110111
11111101110000000001
10101001011000110111
01010111111110110111
01100100001000010001
11101001110110100011
10101000011110001110
10101101001010100111
10001000000000000011
10000101110010101110
00000101000111101011
11110000111010111000
01101101001011100011
10101001000000000010
00011000000101100100
10100001111110000011
11011011110111111111
10101001001111001110
01010101011111101011
00111011001010100011
01110111110001101110
11100001010110000111
00010010101100010110
01001011111011000010
10110111111111110001
11111110111111011110
00110001011110011011
11010001011011101011
10001010011010100011
01111110101011110111
10101101101111011110
00000100011110100110
10111101111011111111
11011001000000000010
01101110110011110010
11101111101010110111
00100001011000101010
01100001101110001111
11001100000001000010
00000101110100010001
01001100110010100111
01000010001000101111
01111101101110010011
11110111001011100110
10010111101000011111
11101001001111100000
00101001100110101100
00000001110001000011
10100101101101111110
00101001111101111111
01110101101110110111
10100000011010001111
00111111111111100110
10100000011011000111
11101100001100101110
00011000000010100010
10111111011101101111
01100111111001101010
00011011111110110110
10101100101111010111
00000100001110000100
11001101011010110101
11000111111110010011
10101001011000100111
10000111110100101010
00000000000010100000
11010000001110110010
10010100101110110111
01111100011101111011
00010100101110101110
00000000001000100011
11100100100000110111
11010101111010101100
11101000101000101110
01000001001000001010
00000001011011101011
10111000001111000010
11011100011011001110
11100001000100101010
00000011000010001100
11101100011010100001
10101100101000010010
11101001101011111111
10101101001111101010
11101000101111111011
00011101101000100110
01000101111111010001
10100101110111111010
10111111101110110011
00110000110110001101
10111111111110001110
10110111101111011111
00000010011111111111
11110001011010010011
00111111111111111011
00001001001000111100
10100100111000110000
11001001000010011110
10001001100010100110
10001000000001000010
10100101011011110111
10000100001011000111
11000010000000101110
11100111111111000110
01000011000110111100
11010001011110000110
10111111111101101011
10011000111010010100
11110111110000110101
11010001111110101001
11010110111110101011
10010101101110111011
10001100011000101111
10111011111111101011
00110110110010100111
10100011111101100111
10001101101101111110
10101001111111111111
10001001001110010111
00101001000110111011
00001000011100100100
00001000110000001001
00111111111011111011
11111011111100111011
00110000011100100111
00010000010010010110
11100001111101000010
10111110111010111110
01010101001101100101
00000110101011100010
10001110111111000100
00000000111011111101
01010101101001100110
11011001110110110111
10111011011110110111
11001010111011001111
10011111011011110111
11011101110010100000
10011011100000001110
10010001011101101110
00000101111000110110
10001001001111110101
10111110111100101101
00101000011111101111
10111101101111110111
10000111111111010011
11000001111110101111
10010001111000100011
00111001111111100111
01010010000010101010
11100100001111001101
11001111000000000100
10001001011010100011
10111111111111111001
11011101000111111111
00011000011010000111
10011111101101110100
11101001111101101111
00110010001010011011
00110011111110101111
01010111101101001111
10101111111110101111
00010100001111111111
11110011111101101100
10110110101011000110
00010001010010011110
10011100010101011001
10010001001001011110
10011001100010011110
10110100011101111001
00110101100111111111
00100100001000001010
11111111111101101111
11111111110111011110
01000001110000010011
11000010011110111010
11000111101110011111
00001001101010110111
10000001101011100111
11001011111111111110
00001001001011000010
01010100101110101001
00110000000000001000
00110001101111110110
