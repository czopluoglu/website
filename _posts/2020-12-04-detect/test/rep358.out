-------------------------------------------------------
                  DETECT SUMMARY OUTPUT
-------------------------------------------------------
 
                    Data File Name: C:/Users/cengiz/Desktop/Github/website/_posts/2020-12-04-detect/test/rep358.dat                                         

              Number of Items used:       20

           Number of Items dropped:        0

               Number of Examinees:      250

                 Minimum Number of 
                Examinees per Cell:        2

         Number of Vectors Mutated:        4

      Maximum Number of Dimensions:       12

                Randomization Seed:    99991

   Minimum percentage of examinees
         used after deleting cells
     having less than  2 examinees:    98.40

   Average percentage of examinees
         used after deleting cells
     having less than  2 examinees:    98.88
 
-------------------------------------------------------

 NUMBER OF DIMENSIONS THAT MAXIMIZE DETECT:  4
 
  Exploratory DETECT Statistics:

              Maximum DETECT value:   0.6557
                   IDN index value:   0.7158
                           Ratio r:   0.6044

 PARTITION WITH MAXIMUM DETECT VALUE:

    1    2    1    2    1    2    2    1    2    2
    3    3    4    3    4    4    3    3    1    4

 CLUSTER MEMBERSHIPS:

   -----------CLUSTER  1-------------

    1    3    5    8   19

   -----------CLUSTER  2-------------

    2    4    6    7    9   10

   -----------CLUSTER  3-------------

   11   12   14   17   18

   -----------CLUSTER  4-------------

   13   15   16   20
 
   ----------------------------------
 
  Covariance Sign Pattern Matrix:
   d+++------++-+------
   +d+++--+-+-++++--+-+
   ++d-----+----+-+-++-
   ++-d+-+--+---++-----
   -+-+d---+--+-++++--+
   -----d+-+++---+-----
   ---+-+d++++---+-----
   -+----+d++-+-+----+-
   --+-++++d++----++---
   -+-+-++++d+-------+-
   +----++-++d-----++-+
   ++--+--+---d++-++-++
   -+---------+d+++++++
   +++++--+---++d++-+--
   -+-++++-----++d+-+--
   --+-+---+--++++d----
   ----+---+-+++---d+++
   -++-------+-+++-+d-+
   --+----+-+-++---+-d+
   -+--+-----+++---+++d
 
 No cross validation for this DETECT run
